docs/api-reference/core.md

```diff
@@ -3436,7 +3436,7 @@ CRIName
 <p>CRIName is a type alias for the CRI name string.</p>
 </p>
 <h3 id="core.gardener.cloud/v1beta1.Capabilities">Capabilities
-(<code>map[string]github.com/gardener/gardener/pkg/apis/core/v1beta1.CapabilityValues</code> alias)</p></h3>
+(<code>map[string]..CapabilityValues</code> alias)</p></h3>
```

Comment:
This change looks unrelated to me, but was produced by `make generate`.

Reply:
I think this is a strange side-effect of your setup. When running the code generator in my setup, it reverts this change.
You can drop this line from the PR so that the CI jobs will pass.